Components/firebase-messaging-42.1001.0.0/samples/FirebaseMessagingQuickstart/FirebaseMessagingQuickstart/Resources/AboutResources.txt

```diff
@@ -0,0 +1,44 @@
+Images, layout descriptions, binary blobs and string dictionaries can be included 
```

Comment:
They automatically get added when you install new packages. In the future, will make sure to gitignore the packages folder